src/main/scala/dpla/ingestion3/enrichments/EnrichmentDriver.scala

```diff
@@ -43,7 +43,16 @@ class EnrichmentDriver(conf: i3Conf) extends Serializable {
       sourceResource = enriched.sourceResource.copy(
         date = enriched.sourceResource.date.map(d => dateEnrichment.parse(d)),
         language = enriched.sourceResource.language.map(l => LanguageMapper.mapLanguage(l)),
-        place = enriched.sourceResource.place.map(p => spatialEnrichment.enrich(p))
+        place = enriched.sourceResource.place.map(p => spatialEnrichment.enrich(p)),
+        `type` = enriched.sourceResource.`type`.map(t => {
```

Comment:
This might be cleaner as a chain of map calls:
```

`type` = enriched.sourceResource.`type`.map(_.toLowerCase)
.map(DcmiTypeMapper.mapDcmiType)
.flatMap(DcmiTypeStringMapper.mapDcmiTypeString)
.map(_.toLowerCase)
```

Reply:
much clearer.